src/lib.rs

```diff
@@ -237,10 +265,25 @@ impl Message {
         }
     }
 
-    pub fn verify(&self, sig: [u8; 65]) -> Result<Vec<u8>, VerificationError> {
-        if !self.valid_now() {
+    pub fn verify(
```

Comment:
can we do some documentation for the public interfaces such as verify etc. especially if parameters are optional?